exdb/tests/integration_tests.py

```diff
@@ -31,14 +34,28 @@ def create_keyword(self, name="Test Keyword"):
     def create_experience(self, exp_status, attendance=0):
         """Creates and returns an experience object with status,
         start_time, end_time and/or name of your choice"""
-        return Experience.objects.get_or_create(author=self.test_user, name="Test Experience", description="test description", start_datetime=self.test_date,
-                                                end_datetime=(self.test_date + timedelta(days=1)), type=self.create_type(), sub_type=self.create_sub_type(), goal="Test Goal", audience="b",
-                                                status=exp_status, attendance=attendance)[0]
+        return Experience.objects.get_or_create(
+            author=self.clients['ra'].user_object,
+            name="Test Experience",
+            description="test description",
+            start_datetime=self.test_date,
+            end_datetime=(self.test_date + timedelta(days=1)),
+            type=self.create_type(),
+            sub_type=self.create_sub_type(),
+            goal="Test Goal",
+            audience="b",
+            status=exp_status,
+            attendance=attendance
+        )[0]
 
     def create_experience_comment(self, exp, message="Test message"):
         """Creates experience comment, must pass an experience"""
         return ExperienceComment.objects.get_or_create(
-            experience=exp, message=message, author=self.test_user, timestamp=self.test_date)[0]
+            experience=exp,
+            message=message,
+            author=self.clients['ra'].user_object,
```

Comment:
I think it is better the way it is.

In short I thought about it and I did it on purpose when I was writing the code.

I didn't do that because I thought that was out of scope. It would follow, if I was aiming for realism, that I would continue to create actual permissions for the hallstaff and a way to figure out if they were actually hallstaff or not. This is even farther out of scope. I would need to update a fair amount of code to deal with the fact that we have all of these different levels of access.

Aside from it being out of scope we also have no way to determine, without actually creating the relevant users and permissions, what the hallstaff access method will need to look like. That reduces this change to literally be changing a string to do precisely the same thing and it has the disadvantage that it removes a good indication that more work is required for permissions. We will know once we add different permissions for accessing a page that the RA's shouldn't really have access and leaving these here will be a good way to find the pages that should really have access restricted to hallstaff once doing that falls in the scope of some task.

The other issue is that this system affects literally every single page that is accessed and to add more work will add more conflicts and time to correct those conflicts. Once this is merged and in use the little tasks necessary to utilize the login system will be distributed and permissions should be a task that conflicts with far less code.